controllers/user.js

```diff
@@ -43,11 +20,8 @@ const getUser = async (req, res) => {
 
 const deleteUser = async (req, res) => {
   const userId = req.params.userId
-  const user = await User.findById(userId).exec()
-
-  if (!user) throw createError(404, USER_NOT_REGISTERED)
 
-  await User.findByIdAndRemove(userId).exec()
+  await userService.deleteUser(userId)
 
   res.status(200).json({ message: 'User deleted.' })
```

Comment:
do you use `message: 'User deleted.'` on ui side? if no, maybe better to use just `res.end()`

Reply:
It was decided to switch to 204 status, without any message, so fixed